osf/management/commands/populate_impact_metrics.py

```diff
@@ -0,0 +1,92 @@
+import datetime as dt
+from random import random
+from django.core.management.base import BaseCommand
+
+from osf.metrics import (
+    InstitutionProjectCounts,
+    PreprintView,
+    PreprintDownload,
+    UserInstitutionProjectCounts,
+)
+
+from osf.models import Institution, OSFUser, Preprint
+
+
+# Edit the following guids based off your local environment
+USERS = [OSFUser.load('abd53u'), OSFUser.load('hy84n'), OSFUser.load('7zyg2')]
+PREPRINTS = [Preprint.load('far32'), Preprint.load('67bzg')]
+INSTITUTION = Institution.load('cos')
+
+def populate_preprint_metrics(dates):
+    for date in dates:
+        for user in USERS:
+            for preprint in PREPRINTS:
+                preprint_view_count = int(10 * random())
+                PreprintView.record_for_preprint(
+                    preprint=preprint,
+                    user=user,
+                    path=preprint.primary_file.path,
+                    timestamp=date,
+                    count=preprint_view_count
```

Comment:
In real data, are these debounced or otherwise combined, or are these generally 1? I suspect the stuff imported from pagecounter had multiple, but new things are probably just 1.

Reply:
These are generally 1. I  opted for this rather than many documents with a count of 1 to decrease the number of requests to ES. I could go either way - I see the logic that many documents with a count of 1 would be more representative of real data even if it takes more requests to ES for the script. What do you think is the preferable choice?

Reply:
Parameters might be nice. An option for how many records, and an option for whether to do one or many per record? There's an upcoming ticket for trying to stress test Impact, and it would be nice to be able to use these to do so.